dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

```diff
@@ -1,511 +1,512 @@
-            if (categoryNotified != null) {
-                for (List<URL> us : categoryNotified.values()) {
-                    for (URL u : us) {
-                        if (buf.length() > 0) {
-                            buf.append(URL_SEPARATOR);
-                        }
-                        buf.append(u.toFullString());
-                    }
-                }
-            }
-            properties.setProperty(url.getServiceKey(), buf.toString());
-            long version = lastCacheChanged.incrementAndGet();
-            if (syncSaveFile) {
-                doSaveProperties(version);
-            } else {
-                registryCacheExecutor.execute(new SaveProperties(version));
-            }
-        } catch (Throwable t) {
-            logger.warn(t.getMessage(), t);
-        }
-    }
-
-    @Override
-    public void destroy() {
-        if (logger.isInfoEnabled()) {
-            logger.info("Destroy registry:" + getUrl());
-        }
-        Set<URL> destroyRegistered = new HashSet<>(getRegistered());
-        if (!destroyRegistered.isEmpty()) {
-            for (URL url : new HashSet<>(getRegistered())) {
-                if (url.getParameter(DYNAMIC_KEY, true)) {
-                    try {
-                        unregister(url);
-                        if (logger.isInfoEnabled()) {
-                            logger.info("Destroy unregister url " + url);
-                        }
-                    } catch (Throwable t) {
-                        logger.warn("Failed to unregister url " + url + " to registry " + getUrl() + " on destroy, cause: " + t.getMessage(), t);
-                    }
-                }
-            }
-        }
-        Map<URL, Set<NotifyListener>> destroySubscribed = new HashMap<>(getSubscribed());
-        if (!destroySubscribed.isEmpty()) {
-            for (Map.Entry<URL, Set<NotifyListener>> entry : destroySubscribed.entrySet()) {
-                URL url = entry.getKey();
-                for (NotifyListener listener : entry.getValue()) {
-                    try {
-                        unsubscribe(url, listener);
-                        if (logger.isInfoEnabled()) {
-                            logger.info("Destroy unsubscribe url " + url);
-                        }
-                    } catch (Throwable t) {
-                        logger.warn("Failed to unsubscribe url " + url + " to registry " + getUrl() + " on destroy, cause: " + t.getMessage(), t);
-                    }
-                }
-            }
-        }
-    }
-
-    @Override
-    public String toString() {
-        return getUrl().toString();
-    }
-
-    private class SaveProperties implements Runnable {
-        private long version;
-
-        private SaveProperties(long version) {
-            this.version = version;
-        }
-
-        @Override
-        public void run() {
-            doSaveProperties(version);
-        }
-    }
-
-}
+/*
```

Comment:
What changes in this file? If no changes for this file, please revert it.

Reply:
I only modified 95 and 96 lines, the local comparison is only modified 95 and 96 lines, I don't know why the whole file is modified here.